frontend/src/fileSender.js

```diff
@@ -54,7 +48,8 @@ class FileSender extends EventEmitter {
           ['encrypt', 'decrypt']
         )
         .catch(err =>
-          console.log('There was an error generating a crypto key')
+          // eslint-disable-next-line no-console
+          console.error('There was an error generating a crypto key')
```

Comment:
Kept this since removing the `catch()` would alter the code paths. This `catch()` is actually swallowing the error, IIUC. So if `window.crypto.subtle.generateKey()` fails for whatever reason, we're intercepting the error and just ignoring it, allowing `Promise.all()` to resolve with potentially an `undefined` or `null` `secretKey`. 

Or not, maybe I'm wrong. 🤷‍♀️